docs/api-reference/deck.md

```diff
@@ -152,6 +152,31 @@ By default, the deck canvas captures all touch interactions. This prop is useful
 
 Extra pixels around the pointer to include while picking. This is helpful when rendered objects are difficult to target, for example irregularly shaped icons, small moving circles or interaction by touch. Default `0`.
 
+#### `getTooltip` (Function, optional)
+
+Function that takes a hovered-over point and displays a tooltip. 
```

Comment:
- Link to the picking info documentation
- Add: if the prop is not specified...